Sources/PostgresKit/Query/PostgresCreateType.swift

```diff
@@ -0,0 +1,48 @@
+/// The `CREATE TYPE` command is used to create a new types in a database.
+///
+/// See `PostgresCreateTypeBuilder`.
+public struct PostgresCreateType: SQLExpression {
+    /// Name of type to create.
+    public var name: SQLExpression
+
+    public var definition: Definition
+
+    public enum Definition {
+//        case composite
```

Comment:
why are these commented out?

Reply:
I really just wanted to put off implementing them while making it apparent why I chose the design I chose here (i.e. why this is a `PostgresCreateType` instead of `PostgresCreateEnum`).

Reply:
Ah okay, that makes sense. Would you mind adding an issue for these two things?